storybook/DeckStory.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Use of this source code is governed by a MIT-style
+ * license that can be found in the LICENSE file or at
+ * https://opensource.org/licenses/MIT.
+ */
+
+import React from "react";
+import { action } from "@storybook/addon-actions";
+import { Card } from "../src/ui/card";
+import { Deck } from "../src/ui/deck";
+
+import { PlayingCard, standardDeck } from "./PlayingCard";
+
+export class DeckStory extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      deck: standardDeck
+    };
+  }
+
+  onClick = () => {
+    const deck = [...this.state.deck];
+    const card = deck.shift();
```

Comment:
I think this behavior should almost certainly be a part of the `Deck` class because it's a very common use case. Maybe we can expose a prop `onClick` in `Deck`? The behavior for when a user clicks on a `Deck` could be:
- remove the top card
- invoke `onClick` and pass the card that was ejected as an argument

Reply:
Excellent, I like that. I can make an example of a card being drawn from the deck, as well as being put back into the deck.